cartography/intel/azure/credentials.py

```diff
@@ -0,0 +1,170 @@
+import logging
```

Comment:
Could we create a folder for `intel/azure/util` and put `credentials.py` in there?

Reply:
moved file to util folder